nncf/experimental/torch2/function_hook/nncf_graph/nncf_graph_builder.py

```diff
@@ -228,17 +230,17 @@ class GraphModelWrapper:
     """
     A class that wraps a PyTorch model with examples inputs and provides an interface
     to build a computational graph of the model.
-
-    :param model: The PyTorch model to be wrapped.
-    :param example_input: A tuple of example input for the model.
     """
 
-    def __init__(self, model: nn.Module, example_input: Any) -> None:
+    def __init__(self, model: nn.Module, example_input: Any, graph: Optional[PTNNCFGraph] = None) -> None:
```

Comment:
I couldn't find the place in the code where `GraphModelWrapper` is created with graph. I would say that it looks like as not use functionality and this generally contradicts the fact that you have a `reset` method.

Reply:
yeap, need to remove it